jxon.js

```diff
@@ -54,11 +54,16 @@
     trueIsEmpty: false,
     autoDate: false,
     ignorePrefixedNodes: false,
-    parseValues: false
+    parseValues: false,
+    forceXmlProlog: false,
+    defaultXmlProlog: '<?xml version="1.0" encoding="UTF-8"?>'
   };
   var aCache = [];
   var rIsNull = /^\s*$/;
   var rIsBool = /^(?:true|false)$/i;
+  var rXmlProlog = /^(<\?xml.*?\?>[\n]?)/;
+  var rXmlPrologAttributes = /\b(version|encoding|standalone)="([^"]+?)"/g;
```

Comment:
these attributes could also be quoted in single quotes `'` according to the [specs](https://www.w3.org/TR/2004/REC-xml-20040204/#sec-prolog-dtd)